use-timescale/migration/about-hypershift.md

```diff
@@ -67,19 +67,30 @@ URIs in the configuration file.
 To see all available configuration options for hypershift, use this command:
 
 ```bash
-docker run -ti timescale/hypershift:0.5 --help
+docker run -ti timescale/hypershift:0.6 --help
 ```
 
+### The hypershift clone tool
+
+You can use the hypershift `clone` command to preserve a copy of your database
```

Comment:
```suggestion
You can use the hypershift `clone` command to make a copy of your database
```
Does `make` works? I think it simplifies the sentence.

Reply:
"make" implies creating something new, which I'm not sure is actually what is happening ...

Reply:
"create" again, means something new.

Reply:
Oh, then lets go with preserve. The reason why I suggested create is Hypershift is responsible to create a copy of the database to the target.